functions/hg/vcs.dirty.fish

```diff
@@ -1,3 +1,3 @@
 function vcs.dirty
-  command hg summary 2>/dev/null | command grep -q 'commit: (clean)' ^/dev/null
+  command hg summary 2> /dev/null | command grep -q 'commit: (clean)' 2> /dev/null
```

Comment:
Hah, half the command used `2>` and the other half used `^`...

Reply:
yeah, there were a couple like that :)